native/cli/Cargo.toml

```diff
@@ -21,6 +21,8 @@ term-table = "1.3.2"
 
 [dependencies.effektio-core]
 path = "../core"
+default-features = false
```

Comment:
Are you sure we want these?

Reply:
![image](https://user-images.githubusercontent.com/48786056/183250848-db66ad7b-6fca-4951-aea2-925adc724d8e.png)
I inserted that code to reset database.
I think anyone would need mock data when he wants to reset his local synapse server.
What do you think?

Reply:
I don't understand. Why did you disabled default features on this dependency?

Reply:
Fixed by https://github.com/effektio/effektio/pull/170/commits/5cad8afaf03ceaadbf8e76c35ab1e5c0257cbc73